text-generation-inference/tests/test_decode_jetstream_quant.py

```diff
@@ -44,8 +44,13 @@ def test_decode_jetstream_quantization(quantization_jetstream_int8, params):
             sequence_length=512,
             expected_text=" Winston Smith,s,s,s,s,s,s,s,s,s,s",
         ),
+        DecodeTestParams(
+            model_id="meta-llama/Llama-3.3-70B-Instruct",
+            sequence_length=1024,
+            expected_text=" Winston Smith, the protagonist of the story, was slowly getting up from bed. He stretched his arms",
+        ),
     ],
-    ids=["Mixtral-8x7B", "Meta-Llama-3-8B" ,"Meta-Llama-3-70B"],
+    ids=["Mixtral-8x7B", "Meta-Llama-3-8B" ,"Meta-Llama-3-70B", "Llama-3.3-70B-Instruct"],
```

Comment:
Only question should we add test for 3.1 and 3.2 or that's unnecessary?

Reply:
We could, but 3.2 is tested with the 1B variant and 3.1 architecture is identical, so I do not think testing it would add any value.